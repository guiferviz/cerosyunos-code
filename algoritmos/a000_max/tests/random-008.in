47
31
667957699 1446047446 148231632 287869904 17433810 1896311994 1435851310 980614788 1258516613 430345337 1052317558 44288954 944329731 1209560454 1640540608 1976031167 1862546774 2120317715 771590446 1219043875 1173978852 205752132 868049435 1564942218 279423536 272779791 556163834 657552238 1300213032 483418700 1918251283
31
1396131426 1565792832 960614826 899497979 756748159 371266182 675434061 638733001 582552011 1770968884 1920155214 1756542421 1083731816 855916084 1428406182 104219155 1648243457 1471430096 52895773 1131569500 2051729631 109344236 1976364122 1032737107 115699530 1814868229 1248409247 1490545727 1831881483 67561882 1221564887
31
813471823 1834734603 297035822 194646164 1323809318 1261804543 803159396 1890553449 1752384600 316074747 1990994718 1214103303 51242849 1975807637 368824386 1525430950 1757591038 1500658040 1104823092 1994718053 1448260044 1045782921 2129819828 1745246560 969333908 677889063 2116739270 78328759 1344365555 1225178915 2111431617
31
2023445592 1964687933 1316856302 1209067115 332954275 659615268 1030207585 159697956 109440028 472220752 1395878220 1566364656 756581500 864433984 454989265 1564244794 1906478997 2039832482 1833820947 1961023561 959072697 914796324 1581223032 784218790 310677597 2038169354 1117832514 1358599994 2075110341 978397159 650926947
31
475340518 1931733067 896309461 68528009 783484130 2065807697 1031284115 2134552364 1317049499 1637510482 1676050338 1925476497 935911923 1535901479 1786693561 641361606 1289890927 1828618885 2065807706 561887553 786613922 1707610763 2116413453 1113716512 1487037140 278678864 551288587 843257519 900355213 282560047 167178009
31
1097826719 1130304736 1416166741 1575128255 594664772 51832688 341017654 1112982402 252514518 288214621 1225486897 1253641861 158881660 1254318831 2050908569 911559426 1097103855 2099064641 418971025 934426155 1861958527 793594704 1166714539 1898842899 1369063650 1871637564 612461936 298709620 1185663656 1075529514 1469918758
31
1099011424 1310929049 991697061 2127144399 1332305689 1481868139 866995912 1882280036 1133027437 663096183 1327266353 271225751 33686331 34992188 1383004821 924746272 2037206611 1569709708 1609176541 1272778492 336485326 1979573981 1134735190 346952999 1659401409 855942519 329623416 801267002 1775434600 1206884772 1019379209
31
460191946 567856050 765734425 176426518 811101701 1666710312 503459424 1747361071 38868365 1295761196 1419246517 1386658880 1297189151 2083593706 1439483093 1858199493 453244099 490033403 1313630693 1676319943 1139010778 1272023976 550216348 1986829671 2137810990 121582996 1396725515 1446659155 1451359133 1819785960 938514638
31
478070535 133117909 2049893481 765879579 1399393442 1898564115 1857253384 156340061 536549741 2081363875 320074430 1042402794 1923403134 1332846642 2046871525 509915929 1748604546 228732803 536358688 389773305 813060232 1577276493 1262640549 420355109 1084291685 1928417240 484104828 1822078283 218282153 2052087478 323321850
31
1124593390 241229593 302895502 1100391645 583241176 2016068278 1343814035 1193559555 242308852 2004334107 199253646 39911859 1471140016 956854959 1990228514 529816693 29298631 1038531223 1036124866 1932144787 1989484089 1665404129 1937972639 521562253 294205917 2095587975 1201146414 910854137 2113453335 476128371 1855320994
31
964850447 1175207509 638481996 847610359 516218034 64027612 1676416038 1698110281 218644871 799524688 374407984 1821701065 552461044 1141292709 1582239896 2053812243 143846230 746221211 391161860 310673846 1190687769 2142261425 56374058 1327334415 2012335632 1988623535 88970377 1210412862 644028138 1489461516 1228162379
31
1376523276 988207221 455215061 15036331 1622804639 1022937614 177086472 1557135523 1643106867 255478401 905033794 1428736309 106785398 1294123345 80925472 343241743 1880367280 74894117 1948879275 1553810112 1139074372 247954184 247110405 1152528922 1087121501 784950699 2041220330 234976558 1782092823 1446458447 1570736826
31
1923714408 1068084472 1009653146 428311958 1200107218 1009981734 1858298983 988937627 299780866 1260953743 556831097 590683102 403381152 1633909046 676341107 316805869 1826556475 2034201503 1675449759 445957834 1273656805 1916774239 612762723 1279415362 133979153 1350412051 463418203 1361889451 210945795 1651937187 952794843
31
1644977147 1156049387 571472678 242188742 687044102 1514976508 1427695789 391072156 526224192 1247759298 1607488004 612531206 1994528304 287019095 1498977654 502747024 1261156260 1938370212 1908657403 1581952388 350206462 378920025 555176095 735654350 1609888790 1255379877 1075946040 1999991675 1486493011 1901388280 974288818
31
186028938 1422992410 1782152440 1330194799 923038965 1916634303 1024037659 1363377861 494348790 151099979 486744282 2135910564 734652735 1835093430 1534079744 1425917019 78743120 1656413067 842602946 959963515 1851014622 1512756154 558091853 443312850 1436931533 190703793 506359379 1139296347 920642043 1015447628 1904552321
31
1154833081 401247018 1005541830 1797407555 1501835693 1333510621 1529406605 534922575 1172889421 1959387884 905781435 1558775166 204816721 1236343123 455577221 167136414 1414167811 101785741 1577616662 482448222 28141086 1470285652 1283715371 1065682918 314309036 1939363773 813047406 187308119 1688545974 54612828 467423999
31
1193407615 1417813823 409321464 1543355331 694601118 1045135193 68798808 674056692 1065262992 521346408 814746793 1509938408 1106576854 133184281 1544249094 1581280318 1772140716 1362721164 1321583801 966163656 2033047978 1082648544 908990846 897047403 780350707 1776255566 1417031840 1664349406 733546119 380502175 1792278122
31
1723464421 1873861222 791739283 1797423312 1902906903 1179521586 450252624 845213093 701853990 694238899 1844727003 1182276678 1664261527 1867634206 297807004 1163533582 1375641791 16847485 1811650960 1355717136 1992967070 1291338914 1370010510 1766111558 1435503333 1107388028 2077918603 1577370582 1682864209 2129703089 1186967005
31
2092274280 1625389743 2016816308 1188263299 1764110409 119306485 175820962 1442953947 800192700 1581809820 2043111711 1586926665 619168209 222707992 980567300 290883946 2071900864 172768240 1111292788 331456466 99934711 793121690 383549322 1101885874 585907071 1982144172 2006095742 427131918 1449711094 447389595 1940588839
31
1888520108 184330629 1302151614 1385453260 1796874922 534405535 801247131 1261678152 772912661 626079866 566077834 1440544746 1799989013 484486473 837867368 898376220 167509099 1126276388 2116762280 811439282 254355050 2130094539 1460724469 712016662 235021603 1859816701 2081668340 1659143694 424192633 1474884822 1906348433
31
1270415342 1494964312 1086622780 398804414 411434184 763550741 1710320569 882973127 1961325192 1963402392 521040154 1740668150 1216244739 1044402081 761945223 1315277552 571532785 315506125 1675569679 828149748 279767021 950659955 548294708 1415455174 951130396 1414837897 526565212 2079400244 399358456 1506010651 1202425156
31
2019453704 924277853 608838649 553729135 933190652 1038032103 590919728 711374924 530999424 275978495 289785651 513919552 826336573 1973319631 1926703781 386165232 94753827 1126231400 514838858 1875504918 433314079 665554698 607440699 1473716366 1492720289 693722953 862211989 1880146023 1481644828 1423236640 1359136066
31
1320064242 538222047 1039276219 256450397 1647337780 591298962 32057034 248879299 449241879 822324355 1408481920 502099733 270627525 1131027067 627882908 947989135 690198052 1488357080 1532082632 1457274208 25263239 1448545549 1767013134 818356974 1347981670 1838776412 2092581057 278840432 1079930127 1567912100 234319805
31
1836267474 1831930348 1583321739 439452484 1865123368 816093899 896037834 265257299 2042415389 290269187 1748581887 896499601 1066542489 1013917660 459870414 142614387 986172260 149328862 1007135588 2118999118 870524026 1811768833 1747942526 1163368713 76645075 782101605 1994149119 1763836694 1096547231 250400006 176055239
31
1420715610 1623301436 1858427197 1426475070 1512442709 382415969 1402974732 1113011187 1866358295 1375050624 1485923022 193094118 1000345690 157962823 1197564435 1848971075 694927116 945170929 1066498327 76735351 2012449932 221696020 1051843618 88654555 1565550633 2105771724 593096689 303614938 1675632687 1495662782 1384900736
31
697531997 140177885 76620875 61907894 651468802 1456191195 2109072544 1002092036 118048337 1138366013 1162519910 643810351 1864985657 967489339 1326055327 249820098 1413800997 602631739 738804626 920586618 1571790027 598348869 1728472309 755951958 1438453577 1711477995 2050740411 639021287 743384691 1068357806 737943697
31
1230775328 58832595 132515365 638184750 1350628175 1208073480 462144273 1419710173 1657545623 619110845 2022026705 681588040 669069716 1321347635 1384355298 1357967092 402631517 2116223248 1559978327 385625900 1345083653 1391085190 1856543876 686973677 1384188507 1538530148 1394738173 542947637 682543823 1102727536 131264770
31
1033585034 2090672042 204438208 1762770883 1463871049 2044600313 867823872 1860086799 1196816745 30879942 1103983873 1617656009 1138869424 760438523 592196140 993370533 1590757449 1291276377 1078289199 1227512084 1741219292 1943583297 1995843401 441865270 869524812 1256688322 1808002505 914299940 1858315128 1979793239 507231487
31
459700392 1306111642 1424473899 1525215473 1841934882 42664453 335222519 1863533399 748912778 236497694 1100061553 736460774 957451504 26134471 1087281412 1084178177 620497052 1583553368 25796586 1846359477 1696999752 742049703 28918429 1189435154 2137997557 1300741621 2087118443 1887342872 1388758345 364006946 55019801
31
264993188 1863933341 1029640246 200659701 1980543248 1838301993 1113616394 518036447 1037462914 1096536527 86621063 841715151 105829711 1582862296 419726391 240361545 998590875 776436140 549496329 1797032151 1092756794 235059023 479099571 1536148990 1569338625 860325873 39763428 1591734776 2075240383 514256666 2047698327
31
23835971 1064369189 2014038459 790689412 1054785489 503976364 128575484 478540784 1994054958 1298250556 255839104 1920214410 1513649292 1606611502 1120263268 630610318 1639476660 30012183 1882442477 1324347966 667704056 1306320364 1918339009 52053603 1853954699 1648142070 714611857 1942685239 137470805 1277505765 1258637816
31
707028568 1695575194 1228983508 1123779936 621401156 990185347 725103648 199311564 287295352 2041175146 2112380877 1200955778 684759609 2113518541 513143537 1014869583 1156279366 400352539 1525523357 2121650790 105292071 111484036 1372984915 760849085 2096277479 59714011 760032103 1057109092 1063856735 527606034 763923711
31
1096523714 915664667 1197711762 503711638 232942128 1731267604 877373502 747245734 698175052 1690176427 683266925 1050352981 1895653650 952421155 190082939 1320152724 1496964674 582565935 115752083 1190387901 951377125 1946319323 1185813785 410120830 913940328 1426418465 1000751867 311678077 1532770620 1237079627 2082708026
31
1016633439 1696985853 1565998832 1717531779 321524976 1430895700 612583731 53659798 1154601500 293481888 726566650 1455058725 1373763258 1128980189 1782711224 1317459077 1634992711 1713347932 202300133 605479270 1170427730 2110672752 706732508 142936953 670206898 1102916115 1090640777 21230628 551508525 126948696 1288160581
31
613208334 1715341149 1968751953 700551845 57984173 1223677875 936493595 603455430 1407051787 1671877949 1918315701 997259129 1837543213 592910367 1752927952 1903375056 1649856129 1055173699 1827751930 1320003404 276864675 326189240 2032585191 1694065725 1267269014 552462171 1974155249 671103392 912244185 1307128978 347748285
31
918467517 1595271149 724672624 167542399 1569608281 1324013546 2006665217 2118638849 502670979 1223542639 1122625626 1506763791 1919235174 1071084506 714522846 972284999 1736245399 372039097 456005808 956777484 440161202 1444786559 1873414285 635677058 784526534 714974277 1742726290 110293982 1789427181 284271847 1300745594
31
299888123 1489301220 55917149 241002878 1764118642 1496559872 1319031896 35208874 983399705 374331954 328974441 372406459 1059298843 1604410323 1356868895 1225559670 1496939717 692816039 1325056918 806682152 1567353809 802994467 1258076836 854832272 2017621447 1827891939 1968079239 1676956547 454540278 1808769653 1831079207
31
1223954535 576446057 2087335315 806570333 2101808459 2081588373 63193586 1005991041 537606117 55677428 971574969 1851334069 333062989 68043892 212808643 353460046 962957217 536900469 956339754 387437480 2134427761 1013299014 941818526 573492576 1046757182 1312897723 984991612 659909163 302861629 970124264 1284248252
31
1292400489 492181717 1679947222 523059075 1805957804 845260459 506694198 1871338987 88972534 1670193063 124654962 40780821 212750993 1537891400 1799282246 741145295 437431991 483587575 579080872 750513288 1432675772 1936519774 199854736 1178958204 2074679153 1135160937 776434236 158725033 772392298 383613844 767978524
31
1748005599 1206107110 367479419 1683208435 2028322161 290805177 1755650917 802726364 2053313408 2124508062 2048238354 445531895 547181991 413884906 1334037013 598131360 980790422 1902677051 1910354858 2026576229 651486032 777028083 1079555301 1941883647 826195723 1941342399 561020834 1103023639 559715759 107646006 319251618
31
1491219360 631865403 1968802009 1747276881 1534092373 1654110659 521514914 1627693698 340603677 771779513 83247800 1589839175 836502871 634938853 506171563 1094751690 419722067 1668475156 231078355 126409795 1313135492 462206799 5381128 997598270 554045194 2033092709 1072358045 1005958647 504201341 857269610 2003324148
31
50437584 628843853 308938219 1619857496 1540180454 634100459 370697448 1682597572 490662137 1334488622 542123207 1158489082 875392705 959375755 919662307 115630785 849682177 31894000 508909048 2132291345 1575831484 1692619634 322395920 223114775 982927166 1620598861 425456950 1625150531 1199579757 1704514749 1476643099
31
36265465 1922858556 1881907072 1550084692 252037885 560101182 898638207 1357634515 2127874097 39071651 729100533 1116932679 2003532663 485065353 1527858769 740931672 430086523 121526269 488834943 1570800690 558033833 1973811413 1885357496 10460795 526217575 1141374230 867136804 871571971 1963556778 225275239 1757726467
31
1433189539 133683175 717439031 985128805 1581102509 1380109069 2071294584 2104182778 1866264532 25285783 1675745060 1189830460 1351428064 559372096 1276102454 992155247 223835143 85168372 1978884698 596346548 1855766621 667889695 1059328106 943103484 1543850143 1222793345 405682204 341800697 885250060 838915573 377524678
31
341865845 726250577 1840796213 275040042 1011579656 2116920623 1141811037 451497629 1266333012 1832040706 2040684007 1184019569 1266556687 277079787 258489729 1466916797 309135770 37877202 941182069 1503284161 47194245 1766866583 2102169826 2100889246 1998004366 2055746008 1816948948 845375865 580672154 1964818928 143941039
31
1636092114 793462223 1630618651 184574298 1957003623 1598115489 500776374 36460782 763066449 2113951189 1718899527 609952896 519013052 1201247135 993502803 1651199976 2106634368 931083184 538156723 1793194136 1562322861 1754123935 595863563 1622342200 870044410 636412700 1562498270 113108601 1669053585 1865544610 1358291430
31
1421607325 1593533112 110383133 472308822 1801270740 1851502982 1129989320 1586659572 1343671656 1890655702 1673665157 465693235 223324339 370622425 888049281 501627722 1690327295 573704355 1221036107 1910617794 555033321 1352276007 250586137 986218136 1485596430 1878196249 1100782376 687571465 1081965336 340231877 1411763541
