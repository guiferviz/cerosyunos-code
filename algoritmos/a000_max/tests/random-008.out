2120317715
2051729631
2129819828
2075110341
2134552364
2099064641
2127144399
2137810990
2081363875
2113453335
2142261425
2041220330
2034201503
1999991675
2135910564
1959387884
2033047978
2129703089
2092274280
2130094539
2079400244
2019453704
2092581057
2118999118
2105771724
2109072544
2116223248
2090672042
2137997557
2075240383
2014038459
2121650790
2082708026
2110672752
2032585191
2118638849
2017621447
2134427761
2074679153
2124508062
2033092709
2132291345
2127874097
2104182778
2116920623
2113951189
1910617794
